apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/FilterPartial.cs

```diff
@@ -92,7 +92,10 @@ public static Filter Or(IEnumerable<Filter> filters)
         /// <summary>
         /// Creates a filter to check that the specified property is not equal to a given value.
         /// </summary>
-        /// <param name="propertyName">The name of the property. Must not be null.</param>
+        /// <param name="propertyName">
+        /// The name of the property. Must not be null.
+        /// <remarks>The argument name is unfortunate and it will be addressed in the next major version release.</remarks>
```

Comment:
Sorry, I wasn't clear before - the `<remarks>` element should be just under `<summary>`, not a child of `<param>`.
Also note that this isn't an argument - it's a parameter.